app/assets/stylesheets/application.scss

```diff
@@ -19,6 +19,7 @@
 @import "font-awesome";
 @import "nav";
 @import "dashboard";
+@import "_footer";
```

Comment:
no need for the `_` you can just do `@import "footer";` its kind of like how we do partials